tests/online_migration/sle12_online_migration/pre_migration.pm

```diff
@@ -24,7 +24,9 @@ sub check_or_install_packages() {
         # check if the packages was installed along with update
         my $output = script_output "rpm -qa yast2-migration zypper-migration-plugin rollback-helper | sort";
         if ($output !~ /rollback-helper.*?yast2-migration.*?zypper-migration-plugin/s) {
-            die "migration packages was not installed along with system update";
+            # record soft failure and install missed packages to continue the test
+            record_soft_failure 'bsc#982150';
+            assert_script_run "zypper -n in yast2-migration zypper-migration-plugin rollback-helper snapper", 190;
```

Comment:
Show me a verification run ;-) Actually, I don't think this will work. `die` does exactly what it says, it *dies* so it does not continue afterwards. I recommend to write the whole block as
```perl
record_soft_failure 'bsc#982150: migration packages were not installed along with system update. Installing missed package to continue the test';
assert_script_run "zypper -n in yast2-migration zypper-migration-plugin rollback-helper snapper", 190;
```

This way there is no comment that might get out of date and the record_soft_failure is expressive why it's there and also what is done as a workaround.